### PR TITLE
{vis}[GCCcore/10.2.0] motif v2.3.8

### PR DESCRIPTION
**Pull request by SimonPinches**
(created using `eb --new-pr`)

### CODE REVIEW

File: easybuild/easyconfigs/m/motif/motif-2.3.8-GCCcore-10.2.0.eb [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+source_urls = [SOURCEFORGE_SOURCE]
+sources = ['%(name)s-%(version)s.tar.gz']
+checksums = ['859b723666eeac7df018209d66045c9853b50b4218cecadb794e2359619ebce7']
+
+builddependencies = [
+    ('Autotools', '20200321'),
+    ('flex', '2.6.4'),
+    ('Bison', '3.7.1'),
+    ('util-linux', '2.36'),
```
Comment (jfgrimm) [inline easybuild/easyconfigs/m/motif/motif-2.3.8-GCCcore-10.2.0.eb:21]:
```suggestion
    ('Bison', '3.7.1'),
    ('binutils', '2.35'),
    ('util-linux', '2.36'),
```

**Review by jfgrimm — DISMISSED** [review]

**Comment by branfosj** [comment]
@boegelbot please test @ generoso

**Review by branfosj — APPROVED** [review]

**Comment by branfosj** [comment]
@boegelbot please test @ jsc-zen2

**Comment by branfosj** [comment]
Test report by @branfosj
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
bear-pg0203u29a.bear.cluster - Linux RHEL 8.6, x86_64, Intel(R) Xeon(R) Platinum 8360Y CPU @ 2.40GHz (icelake), 1 x NVIDIA NVIDIA A100-SXM4-80GB, 520.61.05, Python 3.6.8
See https://gist.github.com/branfosj/59aeaeeebb6d22fcc09c5debb5015313 for a full test report.

**Comment by branfosj** [comment]
@boegelbot please test @ generoso

**Comment by branfosj** [comment]
@boegelbot please test @ jsc-zen2

**Comment by branfosj** [comment]
@boegelbot please test @ jsc-zen2
EB_ARGS="--sanity-check-only"

**Comment by branfosj** [comment]
@boegelbot please test @ jsc-zen2
EB_ARGS="--sanity-check-only"

**Comment by branfosj** [comment]
Going in, thanks @SimonPinches!

**Pull request merged**